docs/user_guides/projects/jobs/notebook_job.md

```diff
@@ -0,0 +1,177 @@
+---
+description: Documentation on how to configure and execute a Jupyter Notebook job on Hopsworks.
+---
+
+# How To Run A Jupyter Notebook Job
+
+## Introduction
+
+All members of a project in Hopsworks can launch the following types of applications through a project's Jobs service:
+
+- Python (*Hopsworks Enterprise only*)
+- Apache Spark
+
+Launching a job of any type is very similar process, what mostly differs between job types is
+the various configuration parameters each job type comes with. After following this guide you will be able to create a Jupyter Notebook job.
+
+!!! note "Kubernetes integration required"
+    Python Jobs are only available if Hopsworks has been integrated with a Kubernetes cluster.
+
+    Hopsworks can be integrated with [Amazon EKS](../../../setup_installation/aws/eks_ecr_integration.md), [Azure AKS](../../../setup_installation/azure/aks_acr_integration.md) and on-premise Kubernetes clusters.
+
+## UI
+
+### Step 1: Jobs overview
+
+The image below shows the Jobs overview page in Hopsworks and is accessed by clicking `Jobs` in the sidebar.
+
+<p align="center">
+  <figure>
+    <img src="../../../../assets/images/guides/jobs/jobs_overview.png" alt="Jobs overview">
+    <figcaption>Jobs overview</figcaption>
+  </figure>
+</p>
+
+### Step 2: Create new job dialog
+
+Click `New Job` and the following dialog will appear.
+
+<p align="center">
+  <figure>
+    <img src="../../../../assets/images/guides/jobs/create_new_job.png" alt="Create new job dialog">
+    <figcaption>Create new job dialog</figcaption>
+  </figure>
+</p>
+
+### Step 3: Set the job type
+
+By default, the dialog will create a Spark job. To instead configure a Jupyter Notebook job, select `PYTHON`.
+
+<p align="center">
+  <figure>
+    <img src="../../../../assets/images/guides/jobs/jobs_select_python.gif" alt="Select Python job type">
+    <figcaption>Select Python job type</figcaption>
+  </figure>
+</p>
+
+### Step 4: Set the script
+
+Next step is to select the Jupyter Notebook to run. You can either select `From project`, if the file was previously uploaded to Hopsworks, or `Upload new file` which lets you select a file from your local filesystem as demonstrated below. By default, the job name is the same as the file name, but you can customize it as shown. 
```

Comment:
Next step is to select ->
Select the ...

(Given you have step numbers, you don't need to write "The next step is to ..."